masterbase/models.py

```diff
@@ -0,0 +1,22 @@
+"""Module of pydantic models."""
+
+from typing import Any
+
+from pydantic import BaseModel
+
+
+class ReportBody(BaseModel):
+    """Report model for report post request body."""
+
+    session_id: str
+    target_steam_id: str
```

Comment:
Fairly certain this is `int` (ISteamUser expects packed Steam64 IDs serialized in decimal).

Reply:
yeah I dont treat them any differently in the codebase

Reply:
but made an int

Reply:
actually kept as a str because it serializes the same way no matter where we use it

Reply:
The point of having it be `int` is to ensure that no one is `POST`ing the wrong type of Steam ID and not knowing why, is there a simpler way to use pydantic to do that?

Reply:
I'll create a separate tracking issue if you feel that's out of scope here, but it'd be great if this could be resolved on this merge

Reply:
I can just cast to a str once validated

Reply:
Excellent! ISteamUser should also provide an additional layer of redundancy regarding validation here. It probably still makes sense to be more consistent, but we hadn't known what type clients would be posting for `steam_id`s at all. It makes sense to follow the precedents set by Valve, so thanks for that.